requirements/constraints.txt

```diff
@@ -18,10 +18,6 @@ alabaster==1.0.0
     # via sphinx
 annotated-types==0.7.0
     # via pydantic
-async-timeout==5.0.1 ; python_version < "3.11"
```

Comment:
Always generate this under the same runtime, don't change it.

Reply:
done with Python 3.10 now